lib/data_grid.dart

```diff
@@ -629,3 +642,47 @@ enum SortingState {
     }
   }
 }
+
+class SortingIconSettings {
+  final IconData ascending;
+  final IconData descending;
+  final double size;
+  final Color? color;
+  final EdgeInsets padding;
+
+  const SortingIconSettings({
+    required this.ascending,
+    required this.descending,
+    required this.size,
+    this.color,
+    required this.padding,
+  });
+
+  const SortingIconSettings.defaults()
```

Comment:
I think we just give the normal constructor default values instead of two constructors

Reply:
Ok will do :)